### PR TITLE
[T4A2][W14-B4]Abel Lim

### PR DESCRIPTION
**Pull request by abellimz**

### CODE REVIEW

**Comment by abellimz** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -25,6 +27,9 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        // For exercise T4A1
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Person.java:30]: Comments should be about the code only. It should still make sense to any future developer reading the comment. If you want to tell something to the reviewer/tutor (but not future developers of the code), use a GitHub comment instead (like this comment I've added).

**Comment by damithc** [comment]
Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -50,7 +50,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```
Comment (damithc) [inline src/seedu/addressbook/parser/Parser.java:53]: Don't forget to change the places where this method is used.

**Pull request closed without merging**